specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -782,6 +782,20 @@
             "type": "string",
             "description": "The name of the dedicated host group."
           },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on the operation.",
+            "enum": [
+              "instanceView"
+            ],
```

Comment:
can we add a description to what additional response is provided with the instanceView expand option.